### PR TITLE
[183] Update home page to be more mobile friendly

### PR DESCRIPTION
**Pull request by chubberlisk**

### CODE REVIEW

File: app/assets/stylesheets/gov.scss [hunk 1]
```diff
@@ -17,10 +17,22 @@
...
+  padding: 0;
 }
 
 .govuk-grid-row .govuk-grid-column-full .govuk-grid-row .govuk-grid-column-one-half:nth-child(even),
 .govuk-grid-row .govuk-grid-column-two-thirds .govuk-grid-row .govuk-grid-column-one-half:nth-child(even) {
-  padding: 0 0 0 15px;
+  padding: 0;
+}
+
+@media (min-width: 40.0625em) {
```
Comment (danielburnley) [inline app/assets/stylesheets/gov.scss]: feels like a weird breakpoint number - we might want to look at what the design-system uses for breakpoints

File: app/assets/stylesheets/gov.scss [hunk 2]
```diff
@@ -17,10 +17,22 @@
 
 .govuk-grid-row .govuk-grid-column-full .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd),
 .govuk-grid-row .govuk-grid-column-two-thirds .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd) {
-  padding: 0 15px 0 0;
+  padding: 0;
```
Comment (danielburnley) [inline app/assets/stylesheets/gov.scss]: you can nest media queries within here which makes it a bit nicer and less duplication, for example: 

```scss
.header {
	background: $black;

	@media only screen and (min-width: 640px) {
		background: $white;
	}
}
```

File: app/assets/stylesheets/gov.scss (hunk 1 above)
Comment (chubberlisk) [reply app/assets/stylesheets/gov.scss]: That's true. I actually took this from inspecting the GovUK frontend CSS on the browser. I'll have a look deeper into the breakpoints it uses in the code instead.

File: app/assets/stylesheets/gov.scss (hunk 2 above)
Comment (chubberlisk) [reply app/assets/stylesheets/gov.scss]: Cool! I didn't know this. Definitely better. I will refactor to this.

Comment (chubberlisk) [reply app/assets/stylesheets/gov.scss]: Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/167/commits/7aed41296d5c9dc4b78f38769d92a21aa653b5c7.

File: app/assets/stylesheets/gov.scss (hunk 1 above)
Comment (chubberlisk) [reply app/assets/stylesheets/gov.scss]: Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/167/commits/7aed41296d5c9dc4b78f38769d92a21aa653b5c7.

File: app/assets/stylesheets/heading.scss [hunk 3]
```diff
@@ -3,7 +3,13 @@
 }
 
 .govuk-heading-xl {
-  font-size: 4rem;
+  font-size: 3rem;
+}
+
+@media (min-width: 40.0625em) {
```
Comment (danielburnley) [inline app/assets/stylesheets/heading.scss]: Be good to have this as above  :eyes:

Comment (chubberlisk) [reply app/assets/stylesheets/heading.scss]: 🤦‍♀ Fixed now!

**Pull request merged**